nemo-wasm/Cargo.toml

```diff
@@ -20,4 +20,5 @@ nemo-physical = { path = "../nemo-physical", default-features = false }
 thiserror = "1.0"
 wasm-bindgen = "0.2"
 wasm-bindgen-futures = "0.4.37"
+wasm-bindgen-test = "0.2"
```

Comment:
shouldn't this be under `[dev-dependencies]`?

Reply:
ah right.